### ISSUE TITLE
Prep dataset for merge demo

### ISSUE THREAD

**Issue by imnasnainaec** [sillsdev/TheCombine#2102]
We would like a video demoing all the things that can be done in MergeDuplicates, so a simple dataset (or set of datasets) that can give clear examples of all the normal situations (including with protected words and senses) is needed.

Some simple examples were created for BVT and can be used as a starting point: [https://drive.google.com/drive/folders/1gEhv_YjZuWiS2qNfCVi29-_UYhaQNEP\_](https://drive.google.com/drive/folders/1gEhv_YjZuWiS2qNfCVi29-_UYhaQNEP_)

**Comment by imnasnainaec** [comment]
Demo bottom bar options (with two sets):
- [ ] Skip: the same set comes up again at the start of the next set
- [ ] Save and continue with no changes: the same set will not show up unless one of the words is edited

Demo basic sense movement
- [ ] Move sense (from a word that has multiple senses) to another word
- [ ] Move sense (from a word that has multiple senses) to the trash
- [ ] Move sense (from a word that has multiple senses) to the empty column to create a new word from it
- [ ] Move final sense of a word (to another word or to the trash) and point out that word is deleted

Demo merging senses & sidebar
- [ ] Move sense (from a word that has multiple senses) into another sense and see the sidebar pop open
- [ ] Use buttons to open & close the sidebar
- [ ] Reorder senses in the sidebar
- [ ] Point out that only the first gloss of merged senses is saved
- [ ] Point out that all semantic domains are saved
- [ ] Drag a sense out of the sidebar to restore it as its own sense

Demo with a protected word
- [ ] Show visual difference when a word is protected
- [ ] Show how, if it has multiple senses, you can move one of them
- [ ] Show how a protected word's final sense cannot be moved

Demo with a protected sense
- [ ] Show visual difference when a sense is protected
- [ ] Show how a protected senses can move
- [ ] Show how a protected sense cannot be deleted
- [ ] Show how a protected sense cannot be merged into another sense
- [ ] Show how a protected sense can have another sense merged into it
- [ ] Show how a protected sense's sidebar cannot have other senses moved above the protected sense

**Comment by imnasnainaec** [comment]
1-Demo basic sense movement

Move sense (from a word that has multiple senses) to another word
Move sense (from a word that has multiple senses) to the trash
Move sense (from a word that has multiple senses) to the empty column to create a new word from it
Move final sense of a word (to another word or to the trash) and point out that word is deleted

...

**Comment by imnasnainaec** [comment]
Planning moved to: https://docs.google.com/document/d/1N2xxXeBbZO6irNeDMCX_90FMn35Ic3WOd4mywttYe40

**Comment by imnasnainaec** [comment]
This data-set works for the demo. A new video will be needed after #2099.
